### PR TITLE
zipping large json files

### PR DESCRIPTION
**Pull request by AlexNoring**
## Fixes
1. Zipped the large json file used to initialize the NGFC example.
2. Removed a duplicate json file `test_NGFC.py` was using for initialization. The flowsheet and pytest file now use the same zipped json.

Resolves issue #352 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by andrewlee94 — APPROVED** [review]
This looks OK to me.

File: idaes/power_generation/flowsheets/NGFC/NGFC_flowsheet.py [hunk 1]
```diff
@@ -1411,15 +1411,15 @@ def main():
     solver = pyo.SolverFactory("ipopt")
     solver.options = {'bound_push': 1e-16}
 
-    if os.path.exists('NGFC_flowsheet_init.json'):
+    if os.path.exists('NGFC_flowsheet_init.gz'):
```
[the remark this answers is not in the document]
Comment (AlexNoring) [reply idaes/power_generation/flowsheets/NGFC/NGFC_flowsheet.py:1414]: `.gz` and `.json.gz` both work. I'm not sure if there is a reason for going with one over the other.

[the remark this answers is not in the document]
Comment (andrewlee94) [reply idaes/power_generation/flowsheets/NGFC/NGFC_flowsheet.py:1414]: One advantage to `json.gz` is that it indicates the correct file type for when it is unzipped.

**Review by andrewlee94 — APPROVED** [review]
Looks good.

**Review by eslickj — APPROVED** [review]

**Pull request merged**